fs/EXT/Ext2.c

```
@@ -118,7 +118,7 @@ int Ext2ReadBlock(uint32_t block, void* buffer) {
     return 0;
 }
 
-static FileSystemDriver ext2_driver = {"EXT2", Ext2Detect, Ext2Mount};
+static FileSystemDriver ext2_driver = {"EXT2", Ext2Detect, Ext2Mount, Ext2Unmount};
```

Clang (14.0.6)

[warning] variable 'ext2_driver' is non-const and globally accessible, consider making it const

(cppcoreguidelines-avoid-non-const-global-variables)